src/components/FileViewer/DataTableViewer.tsx

```
@@ -0,0 +1,833 @@
+  };
+
+  // 创建数据提供者
+  const createDataProvider = async (): Promise<DataProvider> => {
+    switch (fileType) {
+      case 'parquet':
+        const { ParquetDataProvider } = await import('./providers');
+        return new ParquetDataProvider(filePath, fileSize);
+
+      case 'xlsx':
+      case 'ods':
+        const { XlsxDataProvider } = await import('./providers');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)